tests/scripts/thread-cert/node.py

```diff
@@ -1350,6 +1350,11 @@ def srp_client_get_ttl(self) -> int:
     # TREL utilities
     #
 
+    def enable_trel(self):
+        cmd = 'trel enable'
+        self.send_command(cmd)
+        self._expect_done()
+
     def get_trel_state(self) -> Union[None, bool]:
```

Comment:
Not fully related to this PR: Should we rename it as `is_trel_enabled`? It's not very intuitive that `get_trel_state` returns a bool instead of a string representing the state.